Accessibility/VirtualContent/explainer.md

```diff
@@ -97,6 +98,7 @@ function on_scroll_changed() {
     begin_load_next_section();
 
     // Signal to the user that we're loading more content.
+    document.getElementById("main").setAttribute("aria-busy", "true");
```

Comment:
Would it make more sense to set `aria-busy` to `true` right before inserting the response payload, rather than when showing the loading spinner? I'm thinking about the Twitter use case. While the request for new tweets is outstanding, it seems reasonable for ATs to continue paying attention to updates to existing content for things such as retweet count changes.

Reply:
Either case seems like a valid authoring practice to me, since what's important to the AT is to observe the combination of state changes (busy and not) if specified by the author. I'm fine with shuffling this around in the example per your suggestion.

Reply:
Moved per suggestion